packages/now-node-bridge/src/bridge.ts

```diff
@@ -29,6 +29,7 @@ export interface NowProxyResponse {
 }
 
 interface ServerLike {
+  timeout: number;
```

Comment:
Since this is a `ServerLike`, I'd suggest we don't rely on `timeout` being defined:
```suggestion
  timeout?: number;
```

Reply:
Fixed, thanks!